docs/pbc/all/multi-factor-authentication/202505.0/howto-use-multi-factor-authentication-with-glue-api.md

```diff
@@ -0,0 +1,256 @@
+template: howto-guide-template
+last_updated: Apr 7, 2025
+related:
+  - title: Multi-Factor Authentication Feature overview
+    link: docs/pbc/all/multi-factor-authentication/page.version/multi-factor-authentication.html
+  - title: Install the Multi-Factor Authentication feature
+    link: docs/pbc/all/multi-factor-authentication/page.version/install-multi-factor-authentication-feature.html
+  - title: Install Customer Email Multi-Factor Authentication method
+    link: docs/pbc/all/multi-factor-authentication/page.version/install-customer-email-multi-factor-authentication-method.html
+  - title: Create Multi-Factor Authentication methods
+    link: docs/pbc/all/multi-factor-authentication/page.version/create-multi-factor-authentication-methods.html
+---
+
+This guide explains how to activate, deactivate, and use Multi-Factor Authentication (MFA) through Glue API. 
+
+To lean more about MFA methods, see [Multi-Factor Authentication feature overview](/docs/pbc/all/multi-factor-authentication/{{page.version}}/multi-factor-authentication.html).
+
+{% info_block infoBox "MFA Type Used in This Guide" %}
+
+This guide demonstrates the Multi-Factor Authentication (MFA) flow using the email MFA method. If you're implementing other MFA types, adapt the type field in the request payloads accordingly.
+
+{% endinfo_block %}
+
+## Prerequisites
+
+[Install the Multi-Factor Authentication feature](/docs/pbc/all/multi-factor-authentication/{{page.version}}/install-multi-factor-authentication-feature.html)
+[Install Customer Email Multi-Factor Authentication method](/docs/pbc/all/multi-factor-authentication/{{page.version}}/install-customer-email-multi-factor-authentication-method.html)
+
+## 1) Get Available MFA Methods
+
+To determine the user's current MFA status, request the list of available methods.
+
+**Why?**
+
+Before you proceed, you need to know:
+ - whether the user has any MFA methods configured,
+ - and whether those methods are enabled or not yet set up.
+
+**Request**
+
+```http
+GET /multi-factor-auth-types
+Authorization: Bearer <access_token>
+```
+
+**Response**
+
+```json
+{
+  "data": [
+    {
+      "type": "multi-factor-auth-types",
+      "attributes": {
+        "type": "email",
+        "status": 0
+      }
+    }
+  ]
+}
+```
+
+**What the status means:**
+
+| Status | Description               |
+|--------|---------------------------|
+| 0      | Disabled                  |
+| 1      | Activation is in progress |
+| 2      | Enabled                   |
+
+## 2) Activating MFA (First-time Setup)
+
+If the method status is 0 (not enabled), you need to activate MFA.
+
+### 2.1) Trigger activation
+
+This sends a code to the user via the selected method (for example, email).
+
+**Request**
+
+```http
+POST /multi-factor-auth-type-activate
+Authorization: Bearer <access_token>
+Content-Type: application/json
+
+{
+  "data": {
+    "type": "multi-factor-auth-type-activate",
```

Comment:
is it our OOTB behavior that we send this type?

Reply:
As I was informed, yes :)